teams/developer-relations/guide-to-being-a-manager/README.md

```diff
@@ -152,9 +198,114 @@ With regards to this checklist, reviewers should do the following:
 
 #### Stage 4 - For Reviewers Only
 
-* [ ] Assigned modules' folders properly structured within topics
```

Comment:
the checklist should be placed first at the top and then fully explained with sections